src/lib/mixer_module/mixer_module.cpp

```diff
@@ -70,6 +73,12 @@ _control_latency_perf(perf_alloc(PC_ELAPSED, "control latency"))
 #endif
 
 	px4_sem_init(&_lock, 0, 1);
+
+	// Enforce the existence of the test_motor topic, so we won't miss initial publications
```

Comment:
Why would we miss initial publications otherwise? I want to understand that.

Reply:
If the topic does not exist, the subscriber checks for the existence in delayed intervals, which is typically not an issue. We however need to get the data immediately upon first publication here.